notebooks/02_model/baseline_deep_dive.ipynb

```diff
@@ -411,13 +412,33 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "RMSE:\t\t1.054252\n",
-      "MAE:\t\t0.846033\n",
-      "rsquared:\t0.136435\n",
-      "exp var:\t0.136446\n"
+     "ename": "KeyError",
```

Comment:
oopss!

Reply:
Don't know why earlier the notebook was not saved 😭